multiqc/utils/search_patterns.yaml

```diff
@@ -203,6 +203,8 @@ star:
     fn: '*Log.final.out'
 star_genecounts:
     fn: '*ReadsPerGene.out.tab'
+sortmerna:
+    contents: 'Command: sortmerna'
```

Comment:
Not entirely sure actually. E.g.: We tend to rename files based on the tools they have been through, so subsequent logs would have 'sortmerna' in them because the file would be:

`..._sortmerna.fq.gz`

Could match one of the rarer terms in there s.a. `Gumbel lambda =`